crates/admin/src/rest_api/services.rs

```diff
@@ -236,16 +239,26 @@ where
         state: new_state,
     };
 
-    let result = restate_bifrost::append_to_bifrost(
-        &state.bifrost,
-        Arc::new(Envelope::new(
-            create_envelope_header(partition_key),
-            Command::PatchState(patch_state),
-        )),
-    )
-    .await;
+    let envelope = Envelope::new(
+        create_envelope_header(partition_key),
+        Command::PatchState(patch_state),
+    );
+
+    let result = state
+        .ingress
+        .ingest(
+            partition_key,
+            IngestRecord::from_parts(envelope.record_keys(), envelope),
+        )
+        .await
+        .map_err(|err| {
+            warn!("Could not append state patching command to Bifrost: {err}");
+            MetaApiError::Internal(
+                "Failed sending state patching command to the cluster.".to_owned(),
```

Comment:
Is this always true that the patch command could not be appended to Bifrost? What if the ingress is closing after having sent the command out?